http/vibe/http/router.d

```diff
@@ -133,6 +133,12 @@ final class URLRouter : HTTPServerRequestHandler {
 		return this;
 	}
 
+	/// ditto
+	URLRouter match(HTTPMethod method, string path, HTTPServerRequestDelegate handler)
+	{
+		return match!HTTPServerRequestDelegate(method, path, handler);
+	}
```

Comment:
The problem is that the compiler has nothing to infer the argument types of the inline function/delegate in an efficient way. In theory it could determine the common types of all call expressions within the function, but even that would be quite brittle, as the semantics of the template body can in turn depend on the exact delegate type in various ways. In the end it would probably be a combinatoric problem with unpredictable performance characteristics.